### PR TITLE
Add missing fields to generated Deploy methods

### PR DESCRIPTION
**Pull request by jarnaud**
## Motivation

In `ccip/gethwrappers`, the generated `Deploy<contract>` methods do not fill the `address` and `abi` fields.

Eg:

```
func DeployEVM2EVMOffRampHelper(auth *bind.TransactOpts, backend bind.ContractBackend, staticConfig EVM2EVMOffRampStaticConfig, sourceTokens []common.Address, pools []common.Address, rateLimiterConfig RateLimiterConfig) (common.Address, *types.Transaction, *EVM2EVMOffRampHelper, error) {
	parsed, err := EVM2EVMOffRampHelperMetaData.GetAbi()
	if err != nil {
		return common.Address{}, nil, nil, err
	}
	if parsed == nil {
		return common.Address{}, nil, nil, errors.New("GetABI returned nil")
	}

	address, tx, contract, err := bind.DeployContract(auth, *parsed, common.FromHex(EVM2EVMOffRampHelperBin), backend, staticConfig, sourceTokens, pools, rateLimiterConfig)
	if err != nil {
		return common.Address{}, nil, nil, err
	}
	return address, tx, &EVM2EVMOffRampHelper{EVM2EVMOffRampHelperCaller: EVM2EVMOffRampHelperCaller{contract: contract}, EVM2EVMOffRampHelperTransactor: EVM2EVMOffRampHelperTransactor{contract: contract}, EVM2EVMOffRampHelperFilterer: EVM2EVMOffRampHelperFilterer{contract: contract}}, nil
}
``` 

We need to reach this state:

```
func DeployEVM2EVMOffRampHelper(auth *bind.TransactOpts, backend bind.ContractBackend, staticConfig EVM2EVMOffRampStaticConfig, sourceTokens []common.Address, pools []common.Address, rateLimiterConfig RateLimiterConfig) (common.Address, *types.Transaction, *EVM2EVMOffRampHelper, error) {
	parsed, err := EVM2EVMOffRampHelperMetaData.GetAbi()
	if err != nil {
		return common.Address{}, nil, nil, err
	}
	if parsed == nil {
		return common.Address{}, nil, nil, errors.New("GetABI returned nil")
	}

	address, tx, contract, err := bind.DeployContract(auth, *parsed, common.FromHex(EVM2EVMOffRampHelperBin), backend, staticConfig, sourceTokens, pools, rateLimiterConfig)
	if err != nil {
		return common.Address{}, nil, nil, err
	}
	return address, tx, &EVM2EVMOffRampHelper{address: address, abi: *parsed, EVM2EVMOffRampHelperCaller: EVM2EVMOffRampHelperCaller{contract: contract}, EVM2EVMOffRampHelperTransactor: EVM2EVMOffRampHelperTransactor{contract: contract}, EVM2EVMOffRampHelperFilterer: EVM2EVMOffRampHelperFilterer{contract: contract}}, nil
}
```

## Solution

Modify the generator to add the address and abi fields in the returned struct.

### CODE REVIEW

File: core/gethwrappers/abigen.go
Comment (andrevmatos) [inline core/gethwrappers/abigen.go:1]: I believe this kind of change should go in core? https://github.com/smartcontractkit/chainlink

Comment (jarnaud) [reply core/gethwrappers/abigen.go:1]: Indeed it has a company-wide impact, if you think the change make sense I can close this PR and open a new one in core.

Comment (andrevmatos) [reply core/gethwrappers/abigen.go:1]: cc @connorwstein

Comment (jarnaud) [reply core/gethwrappers/abigen.go:1]: Closing this PR as I opened https://github.com/smartcontractkit/chainlink/pull/11061
on the core repository.

**Comment by jarnaud** [comment]
Closing this PR as I opened https://github.com/smartcontractkit/chainlink/pull/11061
on the core repository.

**Pull request closed without merging**